execution/executor-benchmark/src/main.rs

```diff
@@ -324,6 +326,20 @@ enum Command {
 
         #[clap(long, value_parser)]
         checkpoint_dir: PathBuf,
+
```

Comment:
Maybe we want this for commands others than 'RunExecutor' as well.
cc @igor-aptos